src/utility.js

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: MIT
  */
 
-//"use strict";
+"use strict";
```

Comment:
this may make it slower, we should measure that.

Reply:
It shouldn't make it slower, if anything it will hopefully improve performance. And if you ever change to ES modules, then they're automatically in strict mode.